pkg/kubevirt/client.go

```diff
@@ -27,10 +27,22 @@ import (
 )
 
 const (
-	vmiSubresourceURL       = "/apis/subresources.kubevirt.io/%s/namespaces/%s/virtualmachineinstances/%s/%s"
-	annDefaultSnapshotClass = "snapshot.storage.kubernetes.io/is-default-class"
+	vmiSubresourceURL               = "/apis/subresources.kubevirt.io/%s/namespaces/%s/virtualmachineinstances/%s/%s"
+	annDefaultSnapshotClass         = "snapshot.storage.kubernetes.io/is-default-class"
+	InfraStorageClassNameParameter  = "infraStorageClassName"
+	InfraSnapshotClassNameParameter = "infraSnapshotClassName"
 )
 
+type InfraTenantStorageSnapshotMapping struct {
+	VolumeSnapshotClasses []InfraToTenantMapping
+	StorageClasses        []string
+}
+
+type InfraToTenantMapping struct {
+	Infra  string
+	Tenant string
+}
```

Comment:
Yeah I considered that, but as you said the number of entries in here will be pretty low so I didn't think it would make much of a difference, and I thought this was more readable so I preferred that.